pkg/lightning/backend/local/engine.go

```diff
@@ -59,6 +59,9 @@ var (
 	normalIterStartKey = []byte{1}
 )
 
+// DefaultBlockSize ensures we are using a block size larger than 16KB, whereas 4KB is the default block size of Pebble.
```

Comment:
Can we also explain why 4KB pebble default value is not a good choice? and what problem will it cause?